Data/market-hours/market-hours-database.json

```diff
@@ -30964,11 +30949,16 @@
         },
         {
           "start": "09:30:00",
-          "end": "17:00:00",
+          "end": "14:20:00",
           "state": "market"
         },
         {
-          "start": "18:00:00",
+          "start": "14:50:00",
+          "end": "18:00:00",
+          "state": "postmarket"
+        },
+        {
+          "start": "21:00:00",
           "end": "1.00:00:00",
           "state": "postmarket"
```

Comment:
Market starts as 21pm new york until 14:20 of the next day, this should be market, not postmarket? From 14:50 to 18:00 is post market 👌  there's also missing the pre market starting from 19:30 to 21:00 I believe